### DISCUSSION TITLE
Linkplay et Up2stream le multiroom DIY de qualité - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#15] [Comments]
# Linkplay et Up2stream le multiroom DIY de qualité - Haade.fr

Découverte du protocole linkplay probablement l’alternative qualité prix la plus intéressante pour faire du multiroom face à airplay et sonos entre autres, le tout intégré dans homeassistant.

https://haade.fr/fr/blog/decouverte-linkplay-alternative-airplay-sonos

**Comment by xiduis15** [comment]
excellent, je viens d'en installer pour 3 pièces en multiroom. Ça fonctionne au top ! Merci beaucoup pour ce post, les alternatives type yamaha musicast sont tellement chères. Rapport qualié/prix imbattable. Alternative à la boite de dérivation, j'essaye de m'imprimer ce petit case : https://www.thingiverse.com/thing:5975998

**Reply by haade-administrator** [reply]
Cette technologie fonctionne très bien et c'est vrai que la prise en charge dans HA est exemplaire, content que ça te convienne.
